src/scripts/main.js

```diff
@@ -1,5 +1,23 @@
 'use strict';
 
 document.addEventListener('click', (e) => {
-  // write code here
+  const wall = document.querySelector('.wall');
+  const wallRect = wall.getBoundingClientRect();
+
+  const spider = document.querySelector('.spider');
+  const spiderWidth = spider.offsetWidth;
+  const spiderHeight = spider.offsetHeight;
```

Comment:
Ensure that the '.spider' element exists in the DOM. If it doesn't, accessing 'offsetWidth' or 'offsetHeight' will throw an error.